mycroft/stt/__init__.py

```diff
@@ -134,10 +135,29 @@ def execute(self, audio, language=None):
             return self.api.stt(audio.get_flac_data(), self.lang, 1)[0]
 
 
+class MycroftDeepSpeechSTT(STT):
+    """Mycroft Hosted DeepSpeech"""
+    def __init__(self):
+        super(MycroftDeepSpeechSTT, self).__init__()
+        self.api = STTApi("deepspeech")
+
+    def execute(self, audio, language=None):
+        language = language or self.lang
+        if not language.startswith("en"):
+            raise ValueError("Deepspeech is currently english only")
+        try:
+            response = self.api.stt(audio.get_wav_data(), self.lang, 1)
+            return response
+        except exceptions.RequestException as e:
```

Comment:
I think this needs to re-raise the error so the speech client can see if there's a 401 error and should trigger pairing